sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -36,6 +36,13 @@ export interface LogsQueryClientOptions extends PipelineOptions {
    * Defaults to 'https://api.loganalytics.io/v1'.
```

Comment:
Shouldn't this be handled by the service version?

Reply:
That's a good point - might be better to not document it explicitly.

Reply:
Removed the value.